pkg/build/api/types.go

```diff
@@ -63,6 +63,31 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildCreatedEvent is the reason associated with the event registered when a build is created.
+	BuildCreatedEventReason = "BuildCreated"
+	// BuildCreatedEventMessage is the message associatd with the event registered when a build is created.
+	BuildCreatedEventMessage = "Build %s/%s has been created"
+	// BuildStartedEvent is the reason associated with the event registered when a build is started (pod is created).
+	BuildStartedEventReason = "BuildStarted"
+	// BuildStartedEvent is the message associated with the event registered when a build is started (pod is created).
+	BuildStartedEventMessage = "Pod has been created to run build %s/%s"
+	// BuildRunningEvent is the reason associated with the event registered when the build pod starts running.
+	BuildRunningEventReason = "BuildRunning"
+	// BuildRunningEvent is the message associated with the event registered when the build pod starts running.
+	BuildRunningEventMessage = "Pod for build %s/%s started running"
+	// BuildCompletedEvent is the reason associated with the event registered when build completes successfully.
+	BuildCompletedEventReason = "BuildCompleted"
+	// BuildCompletedEvent is the message associated with the event registered when build completes successfully.
+	BuildCompletedEventMessage = "Build %s/%s completed successfully"
+	// BuildFailedEvent is the reason associated with the event registered when build fails.
+	BuildFailedEventReason = "BuildFailed"
+	// BuildFailedEvent is the message associated with the event registered when build fails.
+	BuildFailedEventMessage = "Build %s/%s failed"
+	// BuildCancelledEvent is the reason associated with the event registered when build is cancelled.
+	BuildCancelledEventReason = "BuildCancelled"
```

Comment:
I'm not going to cry about making this simpler, but I do worry that if we provide one kind of event, people are just going to ask why we don't provide the other kind...  ie if we provide build created events, people will say they actually just want to know when the build started, and vice versa.

so again, let's see what @jwforres wants to display in the webconsole and go from there.